config/config.go

```diff
@@ -0,0 +1,73 @@
+package config
+
+import (
+	"flag"
+	"github.com/go-akka/configuration"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var (
+	configPath = flag.String("config", "./config.hocon", "Configuration File Location. Ex: ./config.hocon")
+	conf       *configuration.Config
+	Version    string
+	Bot        bot
+	DB         db
+)
+
+type bot struct {
+	Token  string
+	Prefix string
+}
+
+type db struct {
+	Path string
+}
+
+func Load() error {
```

Comment:
1. `Load` is an exported function, and it should have a comment starting with : 
```
// Load ...
```
2. I would recommend Dependency injection style. This way, the configs will passed into your bot as a parameter ( usually ` configs *Configs`, pointer to reference instead of copy, and to allow `nil` values) .
Blog post about it: https://blog.drewolson.org/dependency-injection-in-go. 
Example in a stable project: [nats.go](https://github.com/nats-io/nats.go/blob/6063d679d23ae5b3edb4030d5e89a6e30ea53bc6/examples/nats-req/main.go#L66). 
Notea bout this method: they used a [Variadic Input function](https://gobyexample.com/variadic-functions) wich allows you to pass multiple variables in the input ( `sum(1, 2) ;  sum(1, 2, 3)`), and used the ellipsis notation, where you spread a list to a variadic input.

Reply:
I'll learn more about DI when MVP is out! For now, documentation added!